ObservatoryCore/Utils/AudioHandler.cs

```diff
@@ -9,6 +10,30 @@ internal class AudioHandler : ConcurrentQueue<KeyValuePair<Guid, string>>
         
         private List<Guid> audioTasks = [];
 
+        public void InstantlyPlay(string filePath)
+        {
+            if (new FileInfo(filePath).Length > 0)
+                try
+                {
+                    using (var file = new AudioFileReader(filePath))
+                    using (var output = new WaveOutEvent() { DeviceNumber = Properties.Core.Default.AudioDevice })
+                    {
+                        output.Init(file);
+                        output.Play();
+                        output.Volume = Properties.Core.Default.AudioVolume;
+
+                        while (output.PlaybackState == PlaybackState.Playing)
+                        {
+                            Thread.Sleep(250);
+                        }
+                    };
+                }
+                catch (Exception ex)
+                {
+                    ErrorReporter.ShowErrorPopup("Audio Playback Error", [(ex.Message, ex.StackTrace ?? string.Empty)]);
+                }
+        }
```

Comment:
This method has not been exposed through `IObservatoryCore` and therefore can't be utilised by any plugins currently.

To minimise interface changes it might be better to do this by overloading `EnqueueAndPlay` or adding a defaulted second parameter to skip the queue.